src/Components/App/App.test.js

```diff
@@ -144,7 +144,7 @@ describe('App', () => {
 
     fireEvent.click(skipIcon)
 
-    const moodRatingHeading = getByRole('heading', { name: /please select a face below/i })
+    const moodRatingHeading = getByRole('heading', { name: /how are you feeling right now?/i })
```

Comment:
I really like this adjustment to the prompt!! This wording seems to invite more introspection!